libethereum/Executive.cpp

```diff
@@ -180,6 +180,26 @@ void Executive::initialize(Transaction const& _transaction)
 {
 	m_t = _transaction;
 
+	try
+	{
+		m_sealEngine.verifyTransaction(ImportRequirements::Everything, _transaction, m_envInfo);
+	}
+	catch (InvalidSignature ex)
```

Comment:
Better catch the exception by const reference, then it's not copied
`catch (InvalidSignature const&)`